src/core/continuous_space.jl

```diff
@@ -180,11 +180,7 @@ end
 
 """
     move_agent!(agent::A, model::ABM{A, ContinuousSpace}, vel::NTuple{D, N}, dt = 1.0)
-Propagate the agent forwards one step according to `vel` and the model's space.
-
-For this continuous space version of `move_agent!`, the "evolution algorithm"
-is a trivial Euler scheme with `dt` the step size, i.e. the agent position is updated
-as `agent.pos += vel * dt`.
+Propagate the agent forwards one step according to `vel`, with dt as the time step,  and the model's space.
 """

```

Comment:
Should it be mentioned that update_vel! isn't called on `vel`? I think it's pretty obvious.

Reply:
It doesn't cost much: I propose to add a simple parenthesis: (`update_vel!` is not used).